### PR TITLE
add tests for nr sacks

### PR DESCRIPTION
**Pull request by yuquanw2**

### CODE REVIEW

File: fuzzer/fuzzer_fragment.c [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #define FUZZ_B_RESERVED2      		(1 << 1)
 #define FUZZ_B_RESERVED3            (1 << 2)
 #define FUZZ_B_RESERVED4      		(1 << 3)
-#define FUZZ_B_RESERVED5            (1 << 4)
+#define NR_SACK_FALG                (1 << 4)
```
Comment (tuexen) [inline fuzzer/fuzzer_fragment.c:12]: Shouldn't this be `NR_SACK_FLAG`?

File: fuzzer/fuzzer_fragment.c [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
 #define FUZZ_B_RESERVED6         	(1 << 5)
 #define I_DATA_FALG                 (1 << 6)
```
Comment (tuexen) [inline fuzzer/fuzzer_fragment.c:14]: and this `I_DATA_FLAG`?

File: fuzzer/fuzzer_fragment.c (hunk 1 above)
Comment (yuquanw2) [reply fuzzer/fuzzer_fragment.c:12]: In corpus, i set first  two bytes to be chunk type. SCTP_NR_SELECTIVE_ACK is 0x10 and corresponding to 1<<4. Are you suggesting me using a different name?

Comment (tuexen) [reply fuzzer/fuzzer_fragment.c:12]: > In corpus, i set first two bytes to be chunk type. SCTP_NR_SELECTIVE_ACK is 0x10 and corresponding to 1<<4. Are you suggesting me using a different name?

I'm only asking whether `FALG` should be `FLAG`? Is this a typo?

Comment (yuquanw2) [reply fuzzer/fuzzer_fragment.c:12]: I am so sorrrry. Yes, this is a typo. Thank you for pointing out. LOL

**Pull request merged**